### ISSUE TITLE
Input .mzXML file, but nothing happened, formular annotation on "MS1 only" supported? 

### ISSUE THREAD

**Issue by xxing9703** [boecker-lab/sirius#34]
I tried to load my .mzXML Data in Sirius, but no peak list was shown on the left side.
LC-MS .raw data was taken on QE, consisting of MS1 scan only, and was converted to .mzXML by ProteoWizard.
I'm wondering if Sirius can annotate MS1 data without fragmentation info provided? and is this (MS1 only) the cause of the problem (not loaded successfully)?

**Comment by mfleisch** [comment]
Hey,
yes we currently do not support MS1 only data and this is also the reason why the import fails.
You should see the messages that explain why your data was not imported in the log window.

We will support this in the future in the command line tool. Support in the GUI might be possible but needs to be discussed.
I will post updates here.

**Comment by xxing9703** [comment]
Hi Markus:  Thanks very much for your quick reply.  We also did MS2 on some of the manually selected features from the MS1 run, and converted into mzXML files.  I tried and however they cannot be loaded successfully either.  Is it because they are not data dependent MS2?  If I want to use Sirius/Zodiac to perform formula annotations of these selected peaks,  how do I need to prepare/convert the data into the right input format?
